YakovlevMV_lab_2/lab 2

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+	setlocale(LC_ALL, "Rus");
+	const int num = 10;
+	int arr[num], min = 32767, max = -32767, nechet[num], kolich = 0;
+	float summ = 0;
+	for (int i = 0; i < num; i++) {
+		arr[i] = rand() % 100;
+		cout << arr[i] << "\t";
+		nechet[i] = -32767;
+	}
+	int i = 1, t = 0;
+	while (i != num) {
+		for (int j = 0; j < num; j++)
+		{
+			if (arr[i] < arr[j])
+			{
+				t = arr[i];
+				arr[i] = arr[j];
+				arr[j] = t;
+			}
+		}
+		i++;
+	}
+	cout << endl;
+	cout << "После сортировки: ";
+	for (int i = 0; i < num; i++) {
+		cout << arr[i] << "\t";
+		
+	}
+	cout << endl;
+	for (int i = 0; i < num; i++) {
+		if (arr[i] % 2 != 0) {
+			nechet[i] = arr[i];
```

Comment:
You should not leave "holes" - values of -32767 in nechet array (or, at least, keep them at the end)